### ISSUE TITLE
Custom ValueConverter<,> fails because of missing using directive in migration designer files

### ISSUE THREAD

**Issue by sayyildiz** [dotnet/efcore#11609]
I have defined and used a custom value converter by extending `ValueConverter<TModel, TProvider>` and it worked well during migration generation. However it failed when I tried to build the project. Because designer files use `ValueConverter<TModel, TProvider>` without declaring the namespace by adding `using Microsoft.EntityFrameworkCore.Storage.Converters;`

Build failed with the following error message.

```
Exception message:
Error	CS0246	The type or namespace name 'ValueConverter<,>' could not be found (are you missing a using directive or an assembly reference?)	
```
### Further technical details
EF Core version: 2.1.0-preview1-final 
Database Provider: Microsoft.EntityFrameworkCore.SqlServer
Operating system: Windows 10
IDE: Visual Studio 2017 15.6.5

**Comment by bricelam** [comment]
I'm not able to repro this. Could you provide your project, or at least the relevant parts of your model code (including the custom value converter)?

**Comment by bricelam** [comment]
Duplicate of #11155

**Comment by sayyildiz** [comment]
I have attached the following sample. 

Key points are:

- The converter which I use drives from ValueConterter<,>
- The converter is defined as `internal`

[EFCore21ConvertersTest.zip](https://github.com/aspnet/EntityFrameworkCore/files/1897536/EFCore21ConvertersTest.zip)

**Comment by bricelam** [comment]
Ah, I see where I went wrong...

This is fixed in 2.1.0-preview2-final
